osis-core/src/main/java/com/scality/osis/service/impl/ScalityOsisService.java

```diff
@@ -123,6 +123,7 @@ public PageOfTenants queryTenants(long offset, long limit, String filter) {
                 return pageOfTenants;
 
             } catch (VaultServiceException e) {
+                logger.error("Query Tenants error. Return empty list. Error details: ", e);
```

Comment:
If no tenant has been created, do we still log the error as the list would be empty?

Reply:
nvm, I understood.